includes/class-sensei-settings.php

```diff
@@ -881,6 +897,29 @@ public function render_learning_mode_setting( $args ) {
 		<?php } ?>
 		<?php
 	}
+
+	/**
+	 * Learning mode setting.
+	 *
+	 * @param array $args The field arguments.
+	 */
+	public function render_woo_commerce_settings( $args ) {
+		$image_path_desktop = Sensei()->assets->get_image( 'purchase-sensei-pro-desktop.png' );
+		$image_path_mobile  = Sensei()->assets->get_image( 'purchase-sensei-pro-mobile.png' );
+		?>
+		<div class="woocommerce-banner">
+			<div class="woocommerce-banner__background woocommerce-banner__background-large woocommerce-banner__background-medium">
+					<span class="woocommerce-banner__header"><?php echo esc_html( __( 'Get sensei pro', 'sensei-lms' ) ); ?></span>
+					<span class="woocommerce-banner__body"><?php echo esc_html( __( 'Sell your courses using the most popular eCommerce platform on the web, WooCommerce.', 'sensei-lms' ) ); ?> </span>
+					<a class="button button-primary woocommerce-banner__redirect-button" href="<?php echo esc_url( 'https://senseilms.com/pricing/' ); ?>"><?php echo esc_html( __( 'Upgrade to Sensei Pro', 'sensei-lms' ) ); ?></a>
+			</div>
+			<div class="woocommerce-banner__side-background">
+				<img class="woocommerce-banner__image-desktop" id="sensei-pricing-image-desktop" src="<?php echo esc_url( $image_path_desktop ); ?>" />
+				<img class="woocommerce-banner__image-mobile" id="sensei-pricing-image-mobile" src="<?php echo esc_url( $image_path_mobile ); ?>" />
```

Comment:
Both images are always loaded using this method. Maybe you could try [srcset](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images)? Here's a quick example:

```
<picture>
	<source media="(max-width:1100px)" srcset="<?php echo esc_url( $image_path_mobile ); ?>">
	<img src="<?php echo esc_url( $image_path_desktop ); ?>" alt="sensei-banner">
</picture>
```

Another option would be to use a CSS `background-image` and change that using media queries.

Reply:
tbh I didn't know about this, thanks !!!

 https://github.com/Automattic/sensei/pull/4801/files#diff-6fc0c94d5794cb7f95469bdef471c292a5d553a8a4664d4b48633a2e7449261dR918